pdf/differential_test/main.py

```diff
@@ -49,6 +50,7 @@ def main():
     rmse_max = collections.defaultdict(lambda: collections.defaultdict(float))
     diff_max = collections.defaultdict(lambda: collections.defaultdict(float))
     any_differential = False
+    temp_dir = tempfile.TemporaryDirectory()
```

Comment:
I think this directory should be passed in the arguments from the FAW. Since the FAW kills subprocesses at some point, any such process (that times out) would leave this folder behind. In contrast, if the FAW controls the temporary folder, then it will delete it after killing the process.